.mergify.yml

```
@@ -76,3 +89,11 @@ pull_request_rules:
     actions:
       queue:
         merge_method: squash
+  - name: rebase and autosquash
+    conditions:
+      - base=master
+      - label=automerge:rebase
+      - -linear-history
```

similarly we should be able to restrict how often this runs to cases not covered by the above

```suggestion
      - -linear-history
      - '#commits-behind=0'
      - -draft
```